REQUIRE

```diff
@@ -1,5 +1,4 @@
 julia 0.4
-FactCheck 0.3
```

Comment:
No, it will be installed automatically when `Pkg.test` is run, and then deleted again. This is the recommended method for test-only dependencies.